CHANGELOG.md

```diff
@@ -50,7 +50,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - Removed `get_node_settings` function as it is no longer used.
   - Renamed `get_customer_id_of_review_host` to `get_customer_id_of_node`.
 - Updated review-database to 0.29.1.
-- In `BlockListSmb`, `Network`, `SamplingPolicy`, `Tidb`, and `TidbRule`,
+- In `BlockListSmb`, `Network`, `SamplingPolicy`, `Tidb`, `TidbRule`,
+  `BlockListConn`, `BlockListDceRpc`, `DnsCovertChannel`, `LockyRansomware`,
+  `CryptocurrencyMiningPool`, `BlockListDns`, `FtpPlainText` AND `BlockListFtp`
```

Comment:
`AND` should be lowercase.

Reply:
I modified it.
I accidentally went into the upper case, but I didn't recognize it.
Thank you for your opinion.